development/generate-attributions.sh

```diff
@@ -38,7 +45,9 @@ main() {
 
   # Check if the script is running in a CI environment (GitHub Actions sets the CI variable to true)
   if [ -z "${CI:-}" ]; then
-    # If not running in CI, restore development dependencies
+    # If not running in CI, restore the allow-scripts plugin and development dependencies.
+    cd "${PROJECT_DIRECTORY}"
```

Comment:
Previously the attempt to re-install development dependencies was also failing, because `yarn` was run in the wrong directory. This is now fixed as well.